containers/hydra/messager.py

```diff
@@ -18,90 +19,108 @@
 #
 #
 # ------------------------------------------------------------------------
-
-import os,sys,argparse
+"""
+import os
+import sys
+import argparse
 import json
 import slack
 from slack import WebClient
 from slack.errors import SlackApiError
 
 
-__version__ = u"0.73300"
-MESSAGETEXT=u""
+__version__ = "0.73300"
+MESSAGETEXT = ""
 
 
 parser = argparse.ArgumentParser(description="Send message to Slack channel (as a Slack app)",
 
 
- epilog="""
+                                 epilog="""
 
     Configuration file content: line0: slack app token, line1:slack channel to be messaged
     Keep token (and configuration file) in safe. Do not store to git.
 
     python3 messager.py  -m "TEXT FOR SLACKCHANNEL" -f CONFIGURATION FILE"""
 
-)
-parser.add_argument('-v', help='Send Slack message', action='version',version="Version:{0}   mika.nokka1@gmail.com ,  MIT licenced ".format(__version__) )
-parser.add_argument("-f",help='<Slack configuration file>',metavar="file")
-parser.add_argument("-m",help='<Slack message>',metavar="message")
+                                 )
+parser.add_argument('-v', help='Send Slack message', action='version',
+                    version=f"Version:{__version__}   mika.nokka1@gmail.com ,  MIT licenced ")
+parser.add_argument("-f", help='<Slack configuration file>', metavar="file")
+parser.add_argument("-m", help='<Slack message>', metavar="message")
```

Comment:
maybe we can add arg validation checks ?

Reply:
Will look this when next change issue under constructions